client/components/category-navigation/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { Button } from '@wordpress/components';
+import { useEffect, useState, useRef } from '@wordpress/element';
+import classnames from 'classnames';
+
+import './style.scss';
+
+type CategoryNavProps = {
+	buttons?: {
+		icon: string;
+		label: string;
+		link: string;
+	}[];
+	list: {
+		name: string;
+		label?: string;
+		link: string;
+	}[];
+	selectedCategory: string;
+};
+
+export const CategoryNavigation = ( { buttons, list, selectedCategory }: CategoryNavProps ) => {
```

Comment:
```suggestion
export const CategoryPillNavigation = ( { buttons, list, selectedCategory }: CategoryPillNavigationProps ) => {
```
I mentioned this here (https://github.com/Automattic/wp-calypso/pull/88244#discussion_r1516143446) as well, but WDYT about tweaking the name of this component like this? `CategoryNavigation` seems quite generic, but including "pill" hints at what the component looks like.

Reply:
I don't like names based on UI, since UI can be changed. But I agree that we should have less generic name, since we already have similar genric components. So I am ok with the proposed name.